src/main/java/net/fabricmc/loader/api/metadata/CustomValue.java

```diff
@@ -110,6 +145,13 @@
 		 * @throws IndexOutOfBoundsException if the index is not within {{@link #size()}}
 		 */
 		CustomValue get(int index);
+
+		/**
+		 * Returns a sequential {@link Stream} with this iterable as its source.
+		 */
+		default Stream<CustomValue> stream() {
```

Comment:
I think the method should be abstract here and implemented in the custom value impl